py/desiutil/test/test_bitmask.py

```diff
@@ -105,6 +106,29 @@ def test_str(self):
             self.assertEqual(bitmask[name].bitnum, self.ccdmask[name].bitnum)
             self.assertEqual(bitmask[name]._extra, self.ccdmask[name]._extra)
 
+    def test_uint64(self):
+        _bitdefs = dict(ccdmask=list())
+        _bitdefs['ccdmask'].append( ['BAD',   0, "badness"] )
+        _bitdefs['ccdmask'].append( ['HOT',   1, "hothot"] )
+        _bitdefs['ccdmask'].append( ['TEST', 16, "testing"] )
+
```

Comment:
Good suggestion.  Implementing that revealed that trying to define the highest bit doesn't work under python 2.7, since the _MaskBit object inherits from python int, which can't store 2**63 in python 2.7 but it can under python 3.5 (where it gracefully becomes a long under the hood).  I added a commented out unittest as a placeholder for debugging that but I suggest that this gets spun off to a different issue and PR and not be a blocking factor for this one.

When we really deprecate python 2.7 this becomes a moot point — mask bits with the highest bit set do work under python 3.5.